### ISSUE TITLE
Problems with shops in the latest version of QB

### ISSUE THREAD

**Issue by SushiScripts** [caticus-mods/QB-Inventory-Theme-Update#5]
When you try to open a store, you get the following error.

SCRIPT ERROR: @qb-shops/server/main.lua:166: No such export CreateShop in resource qb-inventory

**Comment by LettyR** [comment]
Here also, did you find a Way to Fix it ? I really wanna keep the Inventory ...

**Comment by madmechanic187** [comment]
yeah we need a fix its done the same for me none of the shops work in the server

**Comment by caticus-mods** [comment]
The script  was designed and created with default qbcore framework and its 100% working on normal framework. Hundreds of people are using this without issues. Its open source and I do not provide support on free scripts

**Comment by pkerguy96** [comment]
sadly this script stopped working after new update its became now obsolete , if you dont provide support on your free script then might as well delete  this cause its non functional now
